lib/uplink/ext/common_test.go

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"testing"
+
+	"storj.io/storj/internal/testcontext"
+)
+
+// TODO: Start up test planet and call these from bash instead
+func TestCCommonTests(t *testing.T) {
+	ctx := testcontext.New(t)
+	defer ctx.Cleanup()
+
+	planet := startTestPlanet(t, ctx)
+	defer ctx.Check(planet.Shutdown)
+
+	runCTest(t, ctx, "common_test.c")
```

Comment:
Separate tests would end up conflicting here, instead start one testplanet per test.

Reply:
Maybe something like:

```
cfiles, err := filepath.Glob("*_test.c")
require.NoError(t, err)
for _, cfile := range cfiles {
    cfile := cfile
    t.Run(cfile, func(t *testing.T){
        t.Parallel()
        
        ctx := testcontext.New(t)
        defer ctx.Cleanup()

        bin, err := ctx.CompileC(cfile)
        require.NoError(t, err)

        ...
    })
}
```

Reply:
The go tests are more focused on unit tests. The C tests are more focused on integration tests.

Reply:
Based on the code it doesn't look like that, the `unity` thing seems to run unit test like things.